Warn if emcc if not launched via our launcher scripts (#13122)
emscripten-core/emscripten: ChangeLog.md

Current Trunk
-------------

> - Emscripten will now warn if its not run vai the shell/bat launcher scripts.

Reviewer: ```suggestion
- Emscripten will now warn if it is not run via the shell/bat launcher scripts.
```

>   Please use `emcc` rather then `python emcc.py` to run python.  Warning can

Reviewer: ```suggestion
  Please use `emcc` rather then `python emcc.py` to run python.  This warning can
```
